### ISSUE TITLE
Search directly for ORG/CSC number

### ISSUE THREAD

**Issue by anthonyperk** [CurrentlyNotSet/SERB#120]
ORG and CSC request the functionality to type in an ORG number or CSC number directly without going through the advanced search function.

**Comment by parkjohnston** [comment]
this is done....please verify

**Comment by anthonyperk** [comment]
confirmed, waiting on andrew to close

**Comment by parkjohnston** [comment]
that was something i did...i will close

**Comment by parkjohnston** [comment]
Duplicate of #108
